src/features/wallet/SideBarMenu.tsx

```diff
@@ -37,6 +38,7 @@ export function SideBarMenu({
   const disconnects = useDisconnectFns();
   const { readyAccounts } = useAccounts();
   const didMountRef = useRef(false);
+  const sanctioned = useSanctioned();
```

Comment:
Tbh we don't need to check this here. Trevor's first pass PR at a blacklist feature may have misled you.